core/api/src/adapter.rs

```diff
@@ -240,6 +242,61 @@ where
             .ok_or_else(|| APIError::Adapter("Can't find this position".to_string()).into())
     }
 
+    async fn get_proof(
+        &self,
+        _ctx: Context,
+        address: H160,
+        storage_position: Vec<U256>,
+        state_root: Hash,
+    ) -> ProtocolResult<EthAccountProof> {
+        let state_mpt_tree = MPTTrie::from_root(state_root, Arc::clone(&self.trie_db))?;
```

Comment:
This place should use 
https://github.com/axonweb3/axon/blob/cdc29441ee262c5ae03ca59e9bbaa82354cfe298/core/api/src/adapter.rs#L198-L203
The `AxonExecutorReadOnly` contains `get_account` method.

Reply:
`ExecutorContext` is relatively troublesome to obtain. It is easier to use mpt trie directly, because the parameters passed down are only the state root and no corresponding block state.